client/signup/steps/difm-design-picker/demo-tile.tsx

```diff
@@ -0,0 +1,157 @@
+/** @jsxImportSource @emotion/react */
+
+import { Button } from '@automattic/components';
+import { MShotsImage } from '@automattic/design-picker';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+const CustomButton = styled( Button )`
+	opacity: 0;
+	width: 90px;
+	.demo-tile__button-container:hover & {
+		opacity: 1;
+	}
+	transition: 200ms opacity linear, 200ms -webkit-filter linear;
+	@media ( max-width: 960px ) {
+		opacity: 1;
+	}
+`;
+
+export default function DemoTile( {
+	id,
+	selectedDesign,
+	image,
+	templateUrl,
+	name,
+	onShowPreview,
+	selectDesign,
+}: {
+	id: string;
+	selectedDesign: string;
+	image: string;
+	templateUrl: string;
+	name: string;
+	onShowPreview: () => void;
+	selectDesign: () => void;
+} ): JSX.Element {
+	const isSelected = selectedDesign === id;
+	return (
+		<div>
+			<div
+				css={ css`
```

Comment:
👋  Does this do anything? I don't think React base components have a `css` property. Did you mean `style`? (I spotted this because it's a TypeScript error.)

Reply:
Hi! The `css` prop is from Emotion: https://emotion.sh/docs/css-prop

Everything in `client/signup/steps/difm-design-picker/` is slated for deletion since this step is not used anymore, so this won't be an issue. 🙂

Reply:
TIL! I guess there's probably [something tricky](https://github.com/emotion-js/emotion/issues/693) we'd have to do to get that prop recognized by TypeScript. Anyway, thanks for the info! If this will be deleted, then there's no problem. If we need to, we can add `// @ts-nocheck` to the file to ignore the TS errors until then.